src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -1043,6 +1089,14 @@ bool Dataloader::load_defines(DefinitionManager& definition_manager) {
 		Logger::error("Failed to load events!");
 		ret = false;
 	}
+	if (!_load_song_chances(definition_manager, lookup_file(song_chance_file))){
+		Logger::error("Error while loading Song chances!");
+		ret = false;
+	}
+	if (!_load_sfx_defines(definition_manager, lookup_file(sfx_file))){
+		Logger::error("Failed to load sound effect defines");
+		ret = false;
+	}
```

Comment:
```suggestion
	if (!_load_song_chances(definition_manager)) {
		Logger::error("Error while loading Song chances!");
		ret = false;
	}
	if (!_load_sfx_defines(definition_manager)) {
		Logger::error("Failed to load sound effect defines");
		ret = false;
	}
```
These don't need the path arguments, and there should be spaces between the `)`s and `{`s.

Also in order to parse the song chance scripts, this line needs to be added to `Dataloader::parse_scripts` later in this file (with names updated to match any changes made to the struct/header and source names).
```
	PARSE_SCRIPTS("song chance", definition_manager.get_song_chance_manager());
```